spond/spond.py

```diff
@@ -233,6 +251,9 @@ async def get_events(
             (TO DO: probably events for which invites haven't been sent yet?)
             Defaults to False for performance reasons.
             Uses `scheduled` API parameter.
+        response : str, optional
+            Only include events which this response. Valid known options are:
+            `unanswered`, there may be more.
```

Comment:
I think this should read "Only include events with this response *from the authenticated user* ...", to make it clearer that this parameter is not intrinsic to the event itself.